koku/masu/database/provider_billing_source_db_accessor.py

```diff
@@ -54,11 +54,11 @@ def get_uuid(self):
         Args:
             None
         Returns:
-            (String): "UUID v4",
+            (UUID): "UUID v4",
```

Comment:
I believe these public functions returning UUID are still returning a String, correct?

Reply:
Oops, no these are returning uuids now. But you're right, these should continue to return Strings since this is a public function. Fixed~